src/drivers/next-json-cookie/default-cookie-options.ts

```diff
@@ -0,0 +1,10 @@
+import { CookieSerializeOptions } from 'cookie';
+
+const DEFAULT_COOKIE_OPTIONS: CookieSerializeOptions = {
+  ...(process.env.NEXT_PUBLIC_COOKIE_DOMAIN ? { domain: process.env.NEXT_PUBLIC_COOKIE_DOMAIN } : { }),
+  expires: new Date(4102444799000), // Thu 31 Dec 2099 23:59:59
+  sameSite: true,
+  ...(process.env.NEXT_PUBLIC_COOKIE_DOMAIN ? { secure: true } : { }),
+}
```

Comment:
I think this should all be a config object that you pass to a new instance of the `next-json-cookie` driver rather than set in this package.

Reply:
agree